src/alhambra/mixes.py

```diff
@@ -1838,7 +1838,18 @@ def buffer_volume(self) -> Quantity[Decimal]:
         mvol = sum(c.tx_volume(self.total_volume) for c in self.actions)
         return self.total_volume - mvol
 
-    def table(self, tablefmt: TableFormat | str = "pipe", validate: bool = True) -> str:
+    def table(
+        self,
+        tablefmt: TableFormat | str = "pipe",
+        validate: bool = True,
+        floatfmt="g",
```

Comment:
I'm not sure it makes sense to include all the tabulate options here?  Most of them don't do anything for us: we're using strings for every cell, I think, and if we wanted to be careful to align things, we'd probably want to specify that alignment in our code with colalign per column, rather than having the user deal with it.  In other words, we know what the columns are, and how they should be aligned.

Reply:
Actually I don't know what the other `tabulate` options do. I just thought it might be nice to give all the options to configure just in case they want. But I don't actually expect myself to configure most of them.